### PR TITLE
fix `make acceptance`

### PR DESCRIPTION
**Pull request by tbg**
nodes were addressed by host name only, which caused confusion
in environments in which a domain was present (for instance,
apparently CircleCI, as of recently).

DNS entries are now created within the `local` domain instead.

### CODE REVIEW

File: run/local-cluster.sh [hunk 1]
```diff
@@ -75,7 +75,7 @@ DNS_FILE="$DNS_DIR/addn-hosts"
 
 # Start dnsmasq container. We wait in a loop until the DNS additional
 # hosts file appears before starting the dnsmasq process.
-DNS_CID=$(docker run -d -v "$DNS_DIR:/dnsmasq.hosts" --name=$DNSMASQ_NAME $DNSMASQ_IMAGE /bin/sh -c "while true; do if [ -f /dnsmasq.hosts/addn-hosts ]; then break; else echo 'waiting 1s for DNS address info...'; sleep 1; fi; done; /usr/sbin/dnsmasq -d")
+DNS_CID=$(docker run -d -v "$DNS_DIR:/dnsmasq.hosts" --name=$DNSMASQ_NAME $DNSMASQ_IMAGE /bin/sh -c "while true; do if [ -f /dnsmasq.hosts/addn-hosts ]; then break; else echo 'waiting 1s for DNS address info...'; sleep 1; fi; done; cat /dnsmasq.hosts/addn-hosts; cat /etc/resolv.dnsmasq.conf; /usr/sbin/dnsmasq -d")
```
Comment (spencerkimball) [inline run/local-cluster.sh]: Did you intend to leave this output in the script?

**Comment by spencerkimball** [comment]
LGTM. Thanks for fixing this so fast.

File: run/local-cluster.sh (hunk 1 above)
Comment (tschottdorf) [inline run/local-cluster.sh]: You never get to see this unless something goes wrong, in which case you want to see that - so I'd leave it in.

**Comment by tschottdorf** [comment]
There's still something that's odd, but I'm not sure it's something I can fix in this PR. Namely, for some reason, the second node gets node ID = 3 (not 2) and so, of course, node 3 gets 4.
Since the test relies on node i having nodeID i, I had to fiddle with the test loop somewhat.
Ideally I'd undo that change (which I'll point out in the source shortly). What could be the reason for this?

File: run/local-cluster.sh [hunk 2]
```diff
@@ -124,35 +123,36 @@ if [[ $DOCKERHOST != "127.0.0.1" ]]; then
...
   for i in $(seq 1 $NODES); do
-    GOSSIP=$(curl -s $DOCKERHOST:${HTTP_PORTS[$i]}/_status/gossip)
-    for j in $(seq 1 $NODES); do
-      if [[ $(echo $GOSSIP | grep "node-$j") == "" ]]; then
-        ALL_FOUND=0
-        break
+    FOUND_NAMES=""
+    GOSSIP_URL="$DOCKERHOST:${HTTP_PORTS[$i]}/_status/gossip"
+    GOSSIP=$(curl --noproxy '*' -s $GOSSIP_URL)
+    for j in $(seq 1 $((2*NODES))); do
```
Comment (tschottdorf) [inline run/local-cluster.sh]: node the 2*NODES here, added in to make up for the fact that there's a nodeID gap: node1, node3, node4

I should note that this happens reliably on CircleCI, but not locally on my Debian. Could be a timing issue, but I have no clue.

**Comment by spencerkimball** [comment]
Would be good to get output from all processes. I'd be curious to see why we're skipping a NodeID. That might happen if there's some error or a restart...we could increment through a node ID in that event.

**Comment by spencerkimball** [comment]
This _just_ started happening, right? Hmmm. Any luck getting the output from all containers?

**Pull request merged**